### PR TITLE
Fixed URL locations for the user interfaces

### PR DESCRIPTION
**Pull request by aschaeffer**

### CODE REVIEW

File: tool/create_visual_studio2015_project.bat [hunk 1]
```diff
@@ -2,6 +2,10 @@
 cd %~dp0..
 mkdir build
 cd build
+
 conan remote add inexor https://api.bintray.com/conan/inexorgame/inexor-conan --insert
+conan remote add bincrafers https://api.bintray.com/conan/bincrafters/public-conan --insert 3
```
Comment (Croydon) [inline tool/create_visual_studio2015_project.bat]: That is exactly what I told you not to do :)

File: tool/create_visual_studio2015_project.bat [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+conan remote add community https://api.bintray.com/conan/conan-community/conan --insert 4
```
Comment (Croydon) [inline tool/create_visual_studio2015_project.bat]: Please remove

File: tool/create_visual_studio2017_project.bat [hunk 3]
```diff
@@ -2,6 +2,10 @@
 cd %~dp0..
 mkdir build
 cd build
+
 conan remote add inexor https://api.bintray.com/conan/inexorgame/inexor-conan --insert
+conan remote add bincrafers https://api.bintray.com/conan/bincrafters/public-conan --insert 3
```
Comment (Croydon) [inline tool/create_visual_studio2017_project.bat]: Please remove

File: tool/create_visual_studio2017_project.bat [hunk 4]
```diff
@@ -2,6 +2,10 @@
...
+conan remote add community https://api.bintray.com/conan/conan-community/conan --insert 4
```
Comment (Croydon) [inline tool/create_visual_studio2017_project.bat]: Please remove

**Comment by Croydon** [comment]
I guess it would be easier if you just wait for my work/help to get my branch green, as I'm working on fixing this too :)

**Comment by a-teammate** [comment]
> I guess it would be easier if you just wait for my work/help to get my branch green, as I'm working on fixing this too :)

I disagree.
This will take time which isnt needed.
We have two PRs which are mergable but master is broken because of a bug unrelated to us directly.
So we should merge the two PRs and fix it up when your PR is ready :)

Btw: @aschaeffer could you please retest? :D

**Comment by Croydon** [comment]
Master is broken. This is broken. You don't really gain anything from merging this. You want to make merge exceptions for what advantage?

Also if you plan do to this, then you can remove the last two commits again and Hanack doesn't need to retest.

**Comment by a-teammate** [comment]
For the advantage of having a master branch with the readymade features merged.

The "Hanack plz retest" was meant on the other PR. :)

**Comment by a-teammate** [comment]
merged

**Pull request closed without merging**